ui/widgets/menu.js

```diff
@@ -78,6 +78,8 @@ return $.widget( "ui.menu", {
 			// them (focus should always stay on UL during navigation).
 			"mousedown .ui-menu-item": function( event ) {
 				event.preventDefault();
+
+				this._activateItem( event );
```

Comment:
We activate on `mousedown` to handle the scenario of moving the mouse over an item, navigating away from that item with they keyboard, then clicking with the mouse. Since we won't get a `mouseenter` or `mousemove`, we need to activate on `mousedown`. This matches the OS X menu behavior.